command/credits.go

```diff
@@ -0,0 +1,248 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/cli/cli/utils"
+)
+
+var isWindows bool
+
+var thankYou = `
+     _                    _
+    | |                  | |
+_|_ | |     __,   _  _   | |           __
+ |  |/ \   /  |  / |/ |  |/_)   |   | /  \_|   |
+ |_/|   |_/\_/|_/  |  |_/| \_/   \_/|/\__/  \_/|_/
+                                   /|
+                                   \|
+                              _
+                           o | |                           |
+ __   __   _  _  _|_  ,_     | |        _|_  __   ,_    ,  |
+/    /  \_/ |/ |  |  /  |  | |/ \_|   |  |  /  \_/  |  / \_|
+\___/\__/   |  |_/|_/   |_/|_/\_/  \_/|_/|_/\__/    |_/ \/ o
+
+
+`
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Nit: since we only need to randomize seed for `credits` output, can we instead move this line to `credits()` function (and perhaps only to the animation portion)?

This is to reduce doing things in package-level init if possible